components/raftstore-v2/src/worker/refresh_config.rs

```diff
@@ -117,13 +133,88 @@ where
             "to" => self.raft_pool.state.expected_pool_size
         );
     }
+
+    fn resize_apply_pool(&mut self, size: usize) {
+        let current_pool_size = self.apply_pool.get_pool_size();
+        if current_pool_size == size {
+            return;
+        }
+
+        // It may not take effect immediately. See comments of
+        // ThreadPool::scale_workers.
+        // Also, the size will be clamped between min_thread_count and the max_pool_size
+        // set when the apply_pool is initialized. This is fine as max_pool_size
+        // is relatively a large value and there's no use case to set a value
+        // larger than that.
+        self.apply_pool.scale_pool_size(size);
+        let (min_thread_count, max_thread_count) = self.apply_pool.thread_count_limit();
+        if size > max_thread_count || size < min_thread_count {
+            warn!(
+                self.logger,
+                "apply pool scale size is out of bound, and the size is clamped";
+                "size" => size,
+                "min_thread_limit" => min_thread_count,
+                "max_thread_count" => max_thread_count,
+            );
+        } else {
+            info!(
+                self.logger,
+                "resize apply pool";
+                "from" => current_pool_size,
+                "to" => size
+            );
+        }
+    }
+
+    /// Resizes the count of background threads in store_writers.
+    fn resize_store_writers(&mut self, size: usize) {
+        // The resizing of store writers will not directly update the local
+        // cached store writers in each poller. Each poller will timely
+        // correct its local cached in its next `poller.begin()` after
```

Comment:
Here, in this pr, why does `raftstore-v2` not need to update the cached senders in `WriteSenders` in `poller.begin() just like what we do in v1?
https://github.com/tikv/tikv/blob/48b18a66b8a469bea1a5621954b9dd5f078cdb4f/components/raftstore/src/store/fsm/store.rs#L923-L924

Reply:
Good catch. I missed it.